evm/src/cpu/decode.rs

```diff
@@ -49,10 +49,11 @@ const OPCODES: [(u8, usize, bool, usize); 31] = [
     (0x58, 0, false, COL_MAP.op.pc),
     (0x5b, 0, false, COL_MAP.op.jumpdest),
     (0x5f, 0, false, COL_MAP.op.push0),
-    (0x60, 5, false, COL_MAP.op.push),      // 0x60-0x7f
-    (0x80, 4, false, COL_MAP.op.dup),       // 0x80-0x8f
-    (0x90, 4, false, COL_MAP.op.swap),      // 0x90-0x9f
-    (0xf6, 1, true, COL_MAP.op.context_op), // 0xf6-0xf7
+    (0x60, 5, false, COL_MAP.op.push), // 0x60-0x7f
+    (0x80, 4, false, COL_MAP.op.dup),  // 0x80-0x8f
+    (0x90, 4, false, COL_MAP.op.swap), // 0x90-0x9f
+    (0xf6, 1, true, COL_MAP.op.get_context),
+    (0xf7, 1, true, COL_MAP.op.set_context),
```

Comment:
```suggestion
    (0xf6, 0, true, COL_MAP.op.get_context),
    (0xf7, 0, true, COL_MAP.op.set_context),
```
as we split them back, there's no need to check top bits